src/App.tsx

```diff
@@ -1,19 +1,92 @@
-import React from "react";
+import React, { useState, useEffect, useCallback } from "react";
+import { BrowserRouter as Router } from "react-router-dom";
 import "./App.scss";
 import "antd/dist/antd.less";
+import {
+  clearSession,
+  getSession,
+  hasSession,
+  setSession,
+  updateSession,
+} from "./services/session";
 import Header from "./components/Header";
 import Feed from "./components/Feed";
-import Profile from "./components/Profile";
+import ProfileBlock from "./components/ProfileBlock";
```

Comment:
do we like the naming of `ProfileBlock` or is there something better?

Reply:
I think in general, I'd prefer our types to have the proper names. So I think our components may need a slightly different naming convention. We could go with entirely different names but in general that may get confusing.

So I think for now ProfileBlock works for me. If we really want a cleaner name we could go with Account, but again, the alternate names may get confusing. At least ProfileBlock is pretty obvious on it's function.